contracts/contracts/PoolDeposits.sol

```diff
@@ -42,7 +42,12 @@ contract PoolDeposits {
   event DepositWithdrawn(address indexed user);
   event PartialDepositWithdrawn(address indexed user, uint256 amount);
   event ProposalWithdrawn(address indexed benefactor);
-  event InterestSent(address indexed user, uint256 amount, uint256 iteration);
+  event InterestSent(address indexed user, uint256 amount);
```

Comment:
I saw no benefit in including the iteration here for each iteration of the loop (besides increasing gas costs...)

Reply:
Correct here. Wasn't using this value in the graph either.